### PR TITLE
(DOCS-10466, -10467, -11485): Updated System Requirements for Community Windows install.

### PR DESCRIPTION
**Pull request by atsansone**
@rob-mongodb : This updates the [System Requirements](https://docs-mongodbcom-staging.corp.mongodb.com/anthonysansone/DOCS-10466/tutorial/install-mongodb-on-windows.html#system-requirements) for the MongoDB Community install on Windows. A sanity check would be appreciated.

---
This change is [Reviewable](https://reviewable.io/reviews/mongodb/docs/3330)

### CODE REVIEW

File: source/tutorial/install-mongodb-on-windows.txt [hunk 1]
```diff
@@ -15,34 +15,39 @@ Overview
...
+   OS Version:                10.0.17134 N/A Build 17134
+   System Type:               x64-based PC
+
+.. important:: If you are running any edition of Windows Server 2008
+   R2 or Windows 7, please install `a hotfix to resolve an issue with
+   memory mapped files on Windows 
+   <http://support.microsoft.com/kb/2731284>`_.
 
 Get MongoDB Community Edition
 -----------------------------
```
Comment (rob-mongodb) [inline source/tutorial/install-mongodb-on-windows.txt]: Another tech writing guideline: don't immediately follow a heading with another heading. Can we add an introductory sentence between Install MongoDB Community Edition and Interactive Installation?

File: source/tutorial/install-mongodb-on-windows.txt [hunk 2]
```diff
@@ -93,10 +98,11 @@ Manually Create a Windows Service for MongoDB Community Edition
 You can set up the MongoDB server as a :guilabel:`Windows Service` that
 starts automatically at boot time.
 
-The following procedure assumes you have installed MongoDB Community using the
-``.msi`` installer with the path ``C:\Program Files\MongoDB\Server\3.6\``.
+The following procedure assumes you have installed MongoDB Community 
+using the ``.msi`` installer with the path 
+``C:\Program Files\MongoDB\Server\{+version+}\``.
```
Comment (rob-mongodb) [inline source/tutorial/install-mongodb-on-windows.txt]: The trailing backslash is inconsistent with other command line paths in the procedure, such as 'mkdir c:\data\db' in step 2.

File: source/tutorial/install-mongodb-on-windows.txt [hunk 3]
```diff
@@ -15,34 +15,39 @@ Overview
...
+To find which version, service pack, and build of Windows is  
+running on your host and the processor type of that host, enter the 
+following command in either the :guilabel:`Command Prompt` or 
+:guilabel:`Powershell`:
 
 .. code-block:: powershell
 
-   wmic os get caption
-   wmic os get osarchitecture
+   PS C:\> systeminfo | findstr /B /C:"OS Name" /C:"OS Version" /C:"System Type"
```
[the remark this answers is not in the document]
Comment (atsansone) [reply source/tutorial/install-mongodb-on-windows.txt]: I verified that this command works in Command Line and Powershell with @jdestefano-mongo , @steveren and @markbenvenuto . Normally, I would agree about referencing Microsoft docs, but they don't document this clearly anywhere. This was found on a [Windows Blog](https://www.windows-commandline.com/find-windows-os-version-from-command/). Agreed on the formatting, but we don't have differing styles / roles / classes for these. I wish we did.

File: source/tutorial/install-mongodb-on-windows.txt (hunk 1 above)
Comment (atsansone) [reply source/tutorial/install-mongodb-on-windows.txt]: I will clean this up when I go through everything on the last ticket.

File: source/tutorial/install-mongodb-on-windows.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (rob-mongodb) [reply source/tutorial/install-mongodb-on-windows.txt]: I think my trial copy of Windows 10 Enterprise is disabled now that it is expired, which probably explains why my attempt failed.

**Comment by rob-mongodb** [comment]
I don't get the distinction between a file path and a directory path. Which of the following two examples from  this page is a file path and which is a directory path?

-  C:\Program Files\MongoDB\Server\4.0\. (trailing backslash)

- D:\test\mongodb (No trailing backslash)

This still looks inconsistent to me, but I'll lgtm anyway since typing the trailing backslash works on the command line.

**Comment by atsansone** [comment]
@rob-mongodb : I'll make sure all of the paths are fixed so as they are consistent in the end.

Can I please get an approval on this PR?

**Comment by rob-mongodb** [comment]
Reviewed 2 of 7 files at r1.
Review status: all files reviewed at latest revision, 4 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io/reviews/mongodb/docs/3330)*

**Comment by atsansone** [comment]
@kay-kim : RFM for `master`, _should_ be mergeable for `v3.6`.

**Pull request merged**